
#Moteur 1
float32 m1_Period
bool m1_Dir
bool m1_En

#Moteur 2
float32 m2_Period
bool m2_Dir
bool m2_En

#Moteur 3
float32 m3_Period
bool m3_Dir
bool m3_En

#Moteur 4
float32 m4_Period
bool m4_Dir
bool m4_En

bool calib